### ISSUE TITLE
[README] Missing link to the API docs

### ISSUE THREAD

**Issue by aprotyas** [ros2/domain_bridge#65]
There is a broken API docs link in the last paragraph of the package README:

> There is a C++ API that can be integrated into your own process, you can find the [API docs here](https://github.com/ros2/domain_bridge/blob/main/TODO).

Can that link now be replaced with this? https://docs.ros.org/en/ros2_packages/rolling/api/domain_bridge/index.html

**Comment by jacobperron** [comment]
Yes, we can probably point the link to the docs hosted at docs.ros.org.
